### PR TITLE
Register job for swift proxy

### PR DESCRIPTION
**Pull request by kaweue**

### CODE REVIEW

**Comment by kaweue** [comment]
recheck ha

File: pkg/controller/swiftproxy/swiftproxy_register.go [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+	"github.com/Juniper/contrail-operator/pkg/client/keystone"
+	"github.com/Juniper/contrail-operator/pkg/client/kubeproxy"
+	batch "k8s.io/api/batch/v1"
+	core "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
```
Comment (shifali19) [inline pkg/controller/swiftproxy/swiftproxy_register.go:20]: Group imports.

**Comment by kaweue** [comment]
recheck ha

File: pkg/controller/swiftproxy/swiftproxy_register.go (hunk 1 above)
Comment (kaweue) [reply pkg/controller/swiftproxy/swiftproxy_register.go:16]: fixed

**Comment by kaweue** [comment]
recheck ha

**Pull request merged**